docs/Guides/Engineering guides/Guide to the Singularity and Tesla.md

```diff
@@ -2,15 +2,15 @@
 Note: while both the singularity and the tesla are simpler than the [nuclear reactor](Guide-to-the-nuclear-reactor.md), the cost for failure is much higher. Because of this you should be familiar with [engineering](Engineer.md)
 BEFORE touching it, unless, of course, you intend to make the engine fail from the getgo.
  
-## overview
+## Overview
 The singularity and the tesla are very similar to eachother, to the extent that a single guide can comfortably cover both. While they differ in some aspects, such as power generation,
 they can generally be treated the same way. This guide will cover the specifics of both, how to start them, and how to run them.
 
 ## Background
 It is important to know just how the singularity and tesla reactors work before setting them up.
 
 ### The particle accelerator
-The particle accelerator, commonly reffered to as the "pa" is a multi-part machine that draws a large amount of power in order to create streams of high energy (damaging)
+The particle accelerator, commonly reffered to as the "PA" is a multi-part machine that draws a large amount of power in order to create streams of high energy (damaging)
```

Comment:
Hm, I just noticed "reffered" should not have 2 Fs. It should have 2 Rs: "referred."